### PR TITLE
espanso-pre: Add version 2.1.4-beta

### PR DESCRIPTION
**Pull request by tech189**
- Closes #422
- I made two manifests for alpha and beta, alternatively we could have one 'pre-release' manifest that switches to whichever is latest?

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by tech189** [comment]
Don't know why the tests are failing, I removed the trailing whitespace :(

**Comment by rashil2000** [comment]
> alternatively we could have one 'pre-release' manifest that switches to whichever is latest?

This is preferable

File: bucket/espanso-pre.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                    "https://github.com/federico-terzi/espanso/releases/download/v$version/Espanso-Win-Portable-x86_64.zip",
+                    "https://raw.githubusercontent.com/federico-terzi/espanso/v$version/scripts/resources/windows/icon.ico"
+                ],
+                "hash": [
+                    {
+                        "url": "$url.sha256.txt",
+                        "regex": "$sha256"
+                    },
+                    {
+
```
Comment (rashil2000) [inline bucket/espanso-pre.json:52]:
```suggestion
```

Comment (tech189) [reply bucket/espanso-pre.json:52]: It doesn't work, when running checkver, it reforms the json like so:
```json
                    {
                        
                    }
```

Comment (tech189) [reply bucket/espanso-pre.json:52]: If you remove that set of brackets, then checkver tries to get the hash of the icon from https://raw.githubusercontent.com/federico-terzi/espanso/v2.1.4-beta/scripts/resources/windows/icon.ico.sha256.txt which doesn't exist

Comment (rashil2000) [reply bucket/espanso-pre.json:52]: I feel if the icon is not a part of the release asset, we can omit it from the manifest

Comment (tech189) [reply bucket/espanso-pre.json:52]: Agreed, instead I'll submit an issue to add the icon to the portable release.

Comment (tech189) [reply bucket/espanso-pre.json]: Issue is here: https://github.com/federico-terzi/espanso/issues/1025, shall we merge without the icon for now?

Comment (rashil2000) [reply bucket/espanso-pre.json]: Yes

File: bucket/espanso-pre.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        "url": "https://github.com/federico-terzi/espanso/releases",
+        "regex": "v([\\d.]+(?:-alpha|-beta))"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/federico-terzi/espanso/releases/download/v$version/Espanso-Win-Portable-x86_64.zip",
+                "hash": {
+                    "url": "$url.sha256.txt",
+                    "regex": "$sha256"
```
Comment (rashil2000) [inline bucket/espanso-pre.json:38]:
```suggestion
                    "url": "$url.sha256.txt"
```

**Pull request merged**